src/foam/box/socket/SocketConnectionBoxManager.js

```diff
@@ -197,7 +197,16 @@ foam.CLASS({
           box = new SocketConnectionBox(x, key, socket);
           add(box);
           Agency agency = (Agency) x.get(getThreadPoolName());
-          agency.submit(x, (ContextAgent) box, socket.getRemoteSocketAddress().toString());
+          //agency.submit(x, (ContextAgent) box, socket.getRemoteSocketAddress().toString());
```

Comment:
Everything looks good. Can you remove the no longer needed agency code.

Reply:
Will do